src/components/chartArea/chartArea.js

```diff
@@ -94,6 +95,7 @@ class ChartArea extends React.Component {
 
   // ToDo: the domain range needs to be update when additional datasets are added
   getChartDomain({ isXAxisTicks, isYAxisTicks }) {
+    // todo: make this method also account for chart thresholds when calc'ing domain
```

Comment:
need to reconsider this it seems... thresholds will adjust the domain a bit.

Reply:
attempted to incorporate the `dataSetOne.data` and `dataSetOne.thresholds` while calculating domain for now